src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/Contracts/DeleteDatabase.cs

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+#nullable disable
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+using Microsoft.SqlTools.Utility;
+
+namespace Microsoft.SqlTools.ServiceLayer.ObjectManagement.Contracts
+{
+    public class DeleteDatabaseRequestParams : GeneralRequestDetails
```

Comment:
nit: but I still think drop makes sense to use here to keep the same term usage across STS. So it'd just be `DropDatabaseRequestParams`

But not a huge deal if you'd prefer to keep it like this

Reply:
We already have a general Drop method for all object handlers, but I can add an overload for it for databases

Reply:
That would ensure that the same logic is always used even if drop is called too which isn't a bad thing.